src/ApiGateways/Web.Bff.Shopping/aggregator/Models/BasketData.cs

```diff
@@ -1,22 +1,18 @@
-﻿using System.Collections.Generic;
+﻿namespace Microsoft.eShopOnContainers.Web.Shopping.HttpAggregator.Models;
 
-namespace Microsoft.eShopOnContainers.Web.Shopping.HttpAggregator.Models
+public class BasketData
 {
+    public string BuyerId { get; set; }
 
-    public class BasketData
-    {
-        public string BuyerId { get; set; }
-
-        public List<BasketDataItem> Items { get; set; } = new List<BasketDataItem>();
+    public List<BasketDataItem> Items { get; set; } = new List<BasketDataItem>();
```

Comment:
```suggestion
    public List<BasketDataItem> Items { get; set; } = new();
```

Please try to use target-type new expressions when defining object and collection initializers where the type is obvious.

Reply:
Ok, sure